CSharp/BatchExplorer/Models/TaskModel.cs

```diff
@@ -152,16 +179,30 @@ public string NoOutputFilesReason
             }
         }
 
+        private CloudAppendBlob selectedTaskLinkedStorageFile;
```

Comment:
Not all blobs in linked storage will be append blobs. Consider typing instead as OutputFileReference (the File Conventions abstraction) or ICloudBlob (the storage common interface).